frontend/app/controllers/gallery/new.js

```diff
@@ -0,0 +1,16 @@
+import Controller from "@ember/controller"
+
+export default Controller.extend({
+  actions: {
+    createGallery() {
+      let newTitle = this.get("newTitle")
+      let newRecord = this.store.createRecord("album", {
+        title: newTitle
+      })
+      newRecord.save().then(() => {
+        this.transitionToRoute("gallery.index")
+      })
+      this.set("newTitle", "")
+    }
```

Comment:
Avoid using too much variables (`let`) when you don't really use them. Also you reset `newTitle` even though you transition away before it could do anything.

```javascript
createGallery() {
  let newRecord = this.store.createRecord("album", {
    title: this.get("newTitle")
  })
  newRecord.save().then(() => {
    this.transitionToRoute("gallery.index")
  })
}
```